.drone.yml

```diff
@@ -0,0 +1,91 @@
+---
+kind: pipeline
+type: kubernetes
+name: default
+
+trigger:
```

Comment:
We should probably add resource requests that match our other controller.  It will most likely be similar.
```yaml
resources:
  requests:
    cpu: 400
    memory: 2GiB
```